warehouse/admin/views/blacklist.py

```diff
@@ -166,37 +166,17 @@ def add_blacklist(request):
         )
     )
 
-    # Go through and delete anything that we need to delete so that our
-    # blacklist actually blocks things and isn't ignored (since the blacklist
-    # only takes effect on new project registration).
-    # TODO: This should be in a generic function somewhere instead of putting
```

Comment:
One nice to have would be thorough tests around the newly factored `remove_project` function. Right now in both places we call it (`add_blacklist`, and new `delete_project`) our tests are fairly naive.